samples/sensor/vl53l1x/prj.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_I2C=y
+CONFIG_GPIO=y
+CONFIG_SENSOR=y
+CONFIG_VL53L1X_INTERRUPT_MODE=y
+#CONFIG_DEBUG=y
+#CONFIG_LOG=y
+#CONFIG_SENSOR_LOG_LEVEL_DBG=y
```

Comment:
Don't add commented out code. You also don't need `CONFIG_I2C=y` because it gets selected by the driver.